modules/caddyhttp/reverseproxy/healthchecks.go

```diff
@@ -386,7 +386,7 @@ func (h *Handler) doActiveHealthCheck(dialInfo DialInfo, hostAddr string, upstre
 
 	// set headers, using a replacer with only globals (env vars, system info, etc.)
 	repl := caddy.NewReplacer()
-	repl.Set("http.reverse_proxy.active.target_host", hostAddr)
+	repl.Set("http.reverse_proxy.active.target_host", networkAddr)
```

Comment:
It might be worth renaming `target_host` to something else.

Reply:
Yeah I'm thinking either `target_dial` or `target_upstream` :thinking: